docs/ShowInDialogButton.md

```diff
@@ -13,6 +13,7 @@ It can be useful in case you want the ability to show a record linked by a refer
 
 Below is an example of an `<Edit>` view, inside which is a nested `<Datagrid>`, offering the ability to **create**, **edit** and **show** the rows thanks to `<CreateInDialogButton>`, `<EditInDialogButton>` and `<ShowInDialogButton>`:
 
+{% raw %}
```

Comment:
Those markers are needed as soon as you add `{{`, or `}}` in a code sample, because it messes up Jekyll's templating. You can see the problem when you run `make docker-doc`, the console fills with warnings, and the code sample containing the culprit characters doesn't appear in the rendered html pages.